modules/virt-adding-pci-device-hyperconverged-operator.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-adding-pci-device-hyperconverged-operator_{context}"]
+= Adding PCI devices to the HyperConverged Operator
+To enable the availability of PCI devices in the cluster, add details about the PCI devices to the HyperConverged Operator in the YAML file. The PCI devices are exposed and permitted to be used in the cluster.
+
+.Procedure
+. Use the following command to add the PCI device information to the YAML file.
```

Comment:
s/Use/Run (this also appears in other places)

Reply:
This is only one step, so it should be a bullet point (`*`) instead of a numbered list. Alternatively, you could add a step for saving the file or applying it to the cluster, if that is required

Reply:
Updated the content.